svg-to-img.js

```diff
@@ -1,10 +1,10 @@
+var Rsvg = require('librsvg-prebuilt').Rsvg;
```

Comment:
Note that I haven't tested librsvg; although it seems to rely on cairo, which we relied on in the past for font size computation, and that dependency was a pain point to contributions and sysadmin.